analyses/README.md

```diff
@@ -0,0 +1,21 @@
+# How to use analysis modules in the Single cell RNA Seq Snap workflow (ScRNASeqSnap)
+
+This repository contains tools and workflows for analyzing single cell and single nuclei RNA (sc/snRNA) data from 10X sequencing technology. 
+
+We recommend the following order in running analysis modules which is up to customization based on the user's needs.
+1. `fastqc-analysis` module (help=`Required`. Pipeline for FastQC quality control tool for high throughput sequence data analysis.)
+2. `cellranger-analysis` module (help=`Required`. Pipeline for running and summarizing Cell Ranger count for single or multiple libraries.)
+3. `upstream-analysis` module (help=`Required`. Pipeline for estimating QC metrics and filtering low quality cells.)
+4. `integrative-analysis` module (help=`Required`. Pipeline for Integrative analysis.)
+5. `cluster-cell-calling` module (help=`Required`. Pipeline for cluster cell calling and gene marker analysis.)
+6. `cell-contamination-removal-analysis` module (help=`Optional`. To remove clusters and repeat steps (4) and (5), e.g. for PDX experiments)
+7. `cell-types-annotation` module (help=`Required`. Pipeline for annotating cell types.)
```

Comment:
For modules that say help=`Required`, this means that it is a step of the analysis that should always be run, is that correct? If so, I feel like there could be a more informative name than help, maybe something like run=`Required` or analysis=`Required`.

Reply:
I updated the documentation, please let me know how it reads now. See b6d98b3480d36c0488a6c0ddcffba4b5d3d3bcf4.